src/parse/ida.py

```
@@ -0,0 +1,38 @@
+from scipy.stats import gaussian_kde
+
+def parseIda(
+        file_manager, dataset_id, out_simulation, logger=None
+):
+    
+    simulation_data = pd.read_csv(out_simulation, sep='\t')
+    file_manager.store_data(
+        dataset_id, 'simulation_dfs', simulation_data
+    )
+
+    heatmap = simulation_data.loc[:,['monoMasses', 'rt', 'precursorIntensity']]
```

Flake8 (7.2.0)

[error] missing whitespace after ','

(E231)